mcy/src/components/Layout/Layout.jsx

```diff
@@ -1,12 +1,16 @@
 /* eslint-disable react/prop-types */
+
+import { Stack, Typography, styled } from "@mui/material";
 import Footer from "./Footer";
 import Header from "./Header";
 
 const Layout = ({ children }) => {
   return (
-    <div>
+    <>
       <Header />
-    </div>
+
```

Comment:
```suggestion
<ContentWrapper>
{children}
</ContentWrapper>
```
위처럼 영역 구분지어서 children을 받아줘야 레이아웃으로 다른 페이지에서 사용할 수 있음.